source/conf.py

```diff
@@ -258,7 +258,7 @@
 # Grouping the document tree into LaTeX files. List of tuples
 # (source start file, target name, title, author, documentclass [howto/manual]).
 latex_documents = [
-  ('index', 'PerconaMonitoringAndManagement.tex', u'Percona Monitoring and Management Documentation',
+  ('index', 'PerconaMonitoringAndManagement-%s.tex' % release, u'Percona Monitoring and Management Documentation',
```

Comment:
Adds a version suffix to the PDF so I don't have to prior to sending PDF to marketing.